README.md

```diff
@@ -39,7 +39,7 @@ What does it do?
 
 <sup>¹ PyPy is only supported for manylinux wheels.</sup><br>
 <sup>² Windows arm64 support is experimental.</sup><br>
-<sup>³ CPython 3.13 is available using the [`CIBW_PRERELEASE_PYTHONS`](https://cibuildwheel.pypa.io/en/stable/options/#prerelease-pythons) option. Free-threaded mode requires opt-in.</sup><br>
+<sup>³ CPython 3.13 is built by default using Python RCs, starting with cibuildwheel 2.20.</sup><br>
```

Comment:
I believe free-threaded mode will still require opt-in using [CIBW_FREE_THREADED_SUPPORT](https://cibuildwheel.pypa.io/en/stable/options/#free-threaded-support)

Reply:
Good point, added a note!